openbook_posts/tests/views/test_post_comment_replies.py

```diff
@@ -1536,6 +1536,45 @@ def test_replying_on_foreign_post_comment_sends_push_notification_to_commenter(s
             target_user=post_creator
         )
 
+    @mock.patch('openbook_notifications.helpers.send_post_comment_push_notification_with_message')
+    def test_replying_on_foreign_post_comment_sends_push_notification_to_post_creator(self,
```

Comment:
You tested this test failed without your changes right?

Reply:
yep